src/datasources/targeted-messaging/targeted-messaging.datasource.ts

```diff
@@ -135,8 +181,8 @@ export class TargetedMessagingDatasource
     return {
       id: submission.id,
       outreachId: args.targetedSafe.outreachId,
-      targetedSafeId: submission.targeted_safe_id,
-      signerAddress: submission.signer_address,
+      targetedSafeId: args.targetedSafe.id,
+      signerAddress: args.signerAddress,
```

Comment:
As before re. returning the value from the cache/database.

Reply:
Answered in https://github.com/safe-global/safe-client-gateway/pull/2000#discussion_r1792146523